app/main.py

```diff
@@ -1,13 +1,27 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {
+        "line": error["line_number"],
+        "column": error["column_number"],
+        "message": error["text"],
+        "name": error["code"],
+        "source": "flake8"
+    }
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
-    pass
+    return {
+        "errors": [format_linter_error(error) for error in errors],
+        "path": file_path,
+        "status": "failed" if "errors" else "passed"
```

Comment:
```suggestion
        "status": "failed" if errors else "passed"
```
but u don't use it in the last part, so it don't touch the `pytest`
btw, at the last `def` you paste it correct

Reply:
Fix this